### ISSUE TITLE
Can not use chacha20 or salsa20 when I upgrade to 5.2.0

### ISSUE THREAD

**Issue by lixingcong** [shadowsocks/shadowsocks-android#2658]
**Describe the bug**
Can not start when the cipher was set to 'salsa20' or 'chacha20'. I used it normally before I upgrade to 5.2.0.
If you set the cipher to salsa20 or chacha20, It toasts a message```libsslocal exits too fast(exit code:134)```
If you set to another cipher(AES, etc), everything works fine.

**To Reproduce**
1. create a profile as you like
2. set the cipher suite to 'salsa20' or 'chacha20'
3. run it

**Smartphone**
 - Android version: Android 10
 - Device: Oneplus 3T with lineageOS 17.1
 - Version: Google Play store version 5.2.0

**Configuration**
Put an `x` inside the [ ] that applies.

* [x] IPv4 server address
* [ ] IPv6 server address
* [x] Client IPv4 availability
* [ ] Client IPv6 availability
* Encrypt method: salsa20 or chacha20
* Route
  * [ ] All
  * [ ] Bypass LAN
  * [ ] Bypass China
  * [x] Bypass LAN & China
  * [ ] GFW List
  * [ ] China List
  * [ ] Custom rules
* [ ] IPv6 route
* [ ] Apps VPN mode
  * [ ] Bypass mode
* Remote DNS: dns,google
* [ ] DNS over UDP
* Plugin configuration (if applicable):
* [ ] Auto Connect
* [ ] TCP Fast Open
* If you're not using VPN mode, please supply more details here:

**Comment by sophauer** [comment]
https://github.com/shadowsocks/shadowsocks-android/issues/2644

**Comment by lixingcong** [comment]
Thank you and I found salsa20 was not supported in 5.2.0

logcat
```
01-11 17:48:01.751 E/libsslocal(28259): thread 'main' panicked at 'loading config "/data/user/0/com.github.shadowsocks/no_backup/shadowsocks.conf", unsupported method, `salsa20` is not a supported method', bin/sslocal.rs:182:17
```

Please remove salsa20 or chacha20 from the drop down list in the coming release.

**Comment by madeye** [comment]
It should be supported.

Let me check.

**Comment by madeye** [comment]
Okay, only chach20-ietf is supported.

**Comment by madeye** [comment]
Fixed in v5.2.1.
